app_dart/bin/server.dart

```diff
@@ -20,6 +22,9 @@ Future<void> main() async {
       accessTokenProvider: AccessTokenProvider(config),
     );
 
+    final CacheProvider<List<int>> redisCacheProvider = Cache.redisCacheProvider(await config.redisUrl);
```

Comment:
We should probably refactor this into a `CacheService` (`lib/src/service/cache_service.dart`).  Then `CachedRequestHandler` can take a `CacheService` in its constructor rather than a `Cache`.  This will let us abstract away the type of cache and provide it as a `Cache<Uint8List>` out of the box:

```dart
@immutable
class CacheService {
  const CacheService(this. config);

  final Config config;

  Future<Cache<Uint8List>> redisCache() async {
    final CacheProvider<List<int>> provider = Cache.redisCacheProvider(await config.redisUrl);
    return Cache<List<int>>(provider).withCodec<Uint8List>(const _CacheCodec());
  }
}

class _CacheCodec extends Codec<Uint8List, List<int>> {
  const _CacheCodec();

  @override
  Converter<Uint8List, List<int>> get encoder => const _ListIntConverter();

  @override
  Converter<List<int>, Uint8List> get decoder => const _Uint8ListConverter();
}

class _ListIntConverter extends Converter<Uint8List, List<int>> {
  const _ListIntConverter();

  List<int> convert(Uint8List input) => input;
}

class _Uint8ListConverter extends Converter<List<int>, Uint8List> {
  const _Uint8ListConverter();

  Uint8List convert(List<int> input) => Uint8List.fromList(input);
}
```